### PR TITLE
[DRAFT] vdk-frontend: improve npm build stability.

### PR DESCRIPTION
**Pull request by murphp15**

### CODE REVIEW

File: projects/frontend/cicd/install_shared.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@
 echo "Logging npm engines version..."
 npm version
 echo "Installing dependencies..."
-npm install
+npm install --max_old_space_size=1000
```
Comment (murphp15) [inline projects/frontend/cicd/install_shared.sh:8]: @gorankokin 
You want me to put 
```
node --max-old-space-size=1000 `which npm` install
```
Here?

Comment (gorankokin) [reply projects/frontend/cicd/install_shared.sh:8]: yes

**Comment by gorankokin** [comment]
I think this PR is not needed anymore, since we found that k8s kublets are evicting pods due to low ephemeral storage that exceeds requested quota of 0.
Otherwise we found that nodejs v8 engine default heap size limit is 2GB by default, so there isn't any problem for the time being that requires increasing it.
We will monitor ci/cd pipelines in near future and if there is a need for increasing it, we know the way to go.

**Review by ivakoleva — CHANGES_REQUESTED** [review]
I agree, we may close the ticket until further notice

**Pull request closed without merging**